src/python/stup/utils.py

```diff
@@ -23,12 +25,9 @@
 def rand_str(length):
     return ''.join([
         random.choice(string.ascii_letters + string.digits)
-        for i in xrange(length)
+        for i in range(length)
```

Comment:
xrange is faster than range on Python 2 with sequences 10^6 and more
you don't have performance/stress tests with measurements
optimisation without profiling is useless